scripts/build_ffmpeg.sh

```
@@ -384,19 +415,22 @@ then
 
     for arg in $@
     do
-        (build_${arg}_unix $(pwd)/ffmpeg_source ${cores})
-        if [ ${arg} == "nasm" ] || [ ${arg} == "yasm" ]
+        if [[ ${arg} != "--install-dir"* ]]  # Exclude --install-dir from build arguments
         then
-            disable_asm=""
-        fi
+            (build_${arg}_unix $(pwd)/ffmpeg_source ${cores})
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)